src/test/java/com/google/firebase/snippets/FirebaseAuthSnippets.java

```diff
@@ -396,4 +410,191 @@ public Response clearSessionCookieAndRevoke(@CookieParam("session") Cookie cooki
     }
   }
   // [END session_clear_and_revoke]
+
+  public void importUsers() {
+    // [START build_user_list]
+    // Up to 1000 users can be imported at once.
+    List<ImportUserRecord> users = new ArrayList<>();
+    users.add(ImportUserRecord.builder()
+        .setUid("uid1")
+        .setEmail("user1@example.com")
+        .setPasswordHash("passwordHash1".getBytes())
+        .setPasswordSalt("salt1".getBytes())
+        .build());
+    users.add(ImportUserRecord.builder()
+        .setUid("uid2")
+        .setEmail("user2@example.com")
+        .setPasswordHash("passwordHash2".getBytes())
+        .setPasswordSalt("salt2".getBytes())
+        .build());
+    // [END build_user_list]
+
+    // [START import_users]
+    UserImportOptions options = UserImportOptions.withHash(
+        HmacSha256.builder()
+            .setKey("secretKey".getBytes())
+            .build());
+    try {
+      UserImportResult result = FirebaseAuth.getInstance().importUsers(users, options);
+      // The number of successful imports is determined via: result.getSuccessCount().
```

Comment:
I'm gonna suggest that you turn this comments into code. 

```
 UserImportResult result = FirebaseAuth.getInstance().importUsers(users, options);
 int successCount = result.getSuccessCount();
 int failureCount = result. getFailureCount();

System.out.println("...");

for (ErrorInfo indexedError : result.getErrors()) {
 int index = indexedError.getIndex()
 String reason = indexedError.getReason());
 System.out.println("...");
}
```

It probably only make sense to do this for this one snippet. I think you should leave the others as is.

Reply:
Good idea. Done.